paltas/Sampling/sampler.py

```diff
@@ -114,6 +117,101 @@ def sample(self):
 					CROSSOBJECTWARNING = False
 				full_param_dict[component+'_parameters'][param] = (
 					cross_dict[cross_param])
-
+				
```

Comment:
Are these also functions from Padma? For use with the catalog sampler? Same comment as previously, add her github username and write a short authorship note.